beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -672,7 +684,10 @@ impl<'a, T: BeaconChainTypes> FullyVerifiedBlock<'a, T> {
                 let state_root = state.update_tree_hash_cache()?;
 
                 let op = if state.slot % T::EthSpec::slots_per_epoch() == 0 {
-                    StoreOp::PutState(state_root.into(), Cow::Owned(state.clone()))
+                    StoreOp::PutState(
+                        state_root.into(),
+                        Cow::Owned(state.clone_with(CloneConfig::none())),
```

Comment:
This will cause this state to be stored on disk without any committee caches, I wonder if we should preserve the current behaviour and use `CloneConfig::committee_caches_only`?

I also just realised another opportunity for optimisation: when we create a `StorageContainer`, we clone the `BeaconState`, even if it came from an owned `PutState`. I'll try and remove this a push a PR.

Reply:
Nice catch!